examples/diagram-navigation/diagram-navigation/index.html

```diff
@@ -56,37 +56,24 @@
     <section class="container col-10 flex-centered mt-2">
             <section class="col-12 mt-2">
                 <div class="col-12 mb-2">
-                    <h2>Default (no navigation)</h2>
-                    The diagram is too large and does not fit within the container.
-                    <div id="bpmn-container-default" class="bpmn-container"></div>
-                </div>
-                <div class="col-12 mt-40 mb-2">
-                    <h2>With Diagram Navigation enabled</h2>
+                    <h2>Diagram Navigation</h2>
+                    The diagram is too large and does not fit within the container.<br>
                     Diagram navigation is enabled here, you can:
                     <ul>
                         <li>zoom in/out by holding the <kbd>CTRL</kbd> key and rolling the mouse wheel.</li>
                         <li>drag/pan the diagram by holding the mouse left button and moving the mouse to move the diagram within the container.</li>
                     </ul>
```

Comment:
❓ do we want to add information about the macOS CMD (https://github.com/process-analytics/bpmn-visualization-js/issues/773) and touch screen (https://github.com/process-analytics/bpmn-visualization-js/issues/847) support right now?

Reply:
As we specify to use CTRL (need to specify for Windows & Linux OS), I think it would be better to specify also how to use the zoom with MacOS.

Reply:
forget my comment, this will be done when the 2 issues will be implemented for real.